review/src/client/modules/ui/app/app.js

```diff
@@ -0,0 +1,53 @@
+import { LightningElement, track } from 'lwc';
+import {
+    VIEWS,
+    LOGIN_VIEW,
+    SESSION_LIST_VIEW,
+    SESSION_VIEW
+} from 'ui/navigationUtil';
+import { getLoggedInUser, logOut } from 'client/authProvider';
+
+export default class App extends LightningElement {
+    @track loggedUser = undefined;
+    @track view = { name: LOGIN_VIEW };
+
+    connectedCallback() {
+        getLoggedInUser().then(response => {
+            if (response.user_id === undefined) {
+                this.loggedUser = undefined;
+                this.view = { name: LOGIN_VIEW };
```

Comment:
You can set the name of the current view instead of creating a new object everytime the view needs to change.
```suggestion
                this.view.name = LOGIN_VIEW;
```

Reply:
Code got removed when switching to username/password auth flow.